src/components/File.vue

```diff
@@ -94,7 +94,7 @@ export default {
 			return this.item.injected.mime.startsWith('image')
 		},
 		src() {
-			return generateUrl(`/core/preview?fileId=${this.item.injected.fileid}&x=${256}&y=${256}&a=${!this.croppedLayout}&v=${this.item.injected.etag}`)
+			return generateUrl(`/core/preview?fileId=${this.item.injected.fileid}&c=${this.item.injected.etag.replace('&quot;', '').replace('&quot;', '')}&x=${250}&y=${250}&forceIcon=0&a=${this.croppedLayout ? '0' : '1'}`)
```

Comment:
This should be a method or something :grin: 

`this.item.injected.etag.replace('&quot;', '').replace('&quot;', '')`